prov/verbs/src/fi_verbs.h

```diff
@@ -730,16 +735,24 @@ int fi_ibv_get_rdma_rai(const char *node, const char *service, uint64_t flags,
 struct verbs_ep_domain {
 	char			*suffix;
 	enum fi_ep_type		type;
+	uint32_t		protocol;
 	uint64_t		caps;
 };
 
 extern const struct verbs_ep_domain verbs_dgram_domain;
+extern const struct verbs_ep_domain verbs_msg_xrc_domain;
 
 int fi_ibv_check_ep_attr(const struct fi_info *hints,
 			 const struct fi_info *info);
 int fi_ibv_check_rx_attr(const struct fi_rx_attr *attr,
 			 const struct fi_info *hints,
 			 const struct fi_info *info);
+static inline int fi_ibv_cmp_xrc_domain_name(const char *domain_name,
+					     const char *rdma_name)
+{
+	return strncmp(domain_name, rdma_name, strlen(domain_name) -
+		       strlen(verbs_msg_xrc_domain.suffix));
```

Comment:
Need to check that strlen(domain_name) > strlen(verbs_msg_xrc_domain.suffix)

Reply:
Right, thanks. I've updated the function.